install/check-install-version.md

```diff
@@ -0,0 +1,33 @@
+# Checking Knative Installation Version
```

Comment:
I tried to run this command with only build installed (and know it says for serving in the first paragraph) but think that we should be clearer with the title too? ("Knative Serving")